frontend/src/app.tsx

```diff
@@ -8,6 +8,21 @@ import { NewDocumentPage } from './pages/new_document';
 import { DocumentPage } from './pages/document';
 import { PageNotFoundPage } from './pages/page_not_found';
 import { ModalHolder } from './components/modal';
+import { SWRConfig, Cache, State } from 'swr';
+
+function localStorageProvider(): Cache {
```

Comment:
Do we really want that? This potentialy leaks data to unauthorized sessions. At least we should clear the cache when logging out or beeing logged out.

Reply:
Probably not. I'll remove it. This was meant as a quick fix because the server currently is very slow (every request takes >1s), but we should fix this in the server, not workaround in the client